src/neoxp/Node/Modules/ExpressRpcMethods.cs

```diff
@@ -3,18 +3,24 @@
 using System.Linq;
 using System.Numerics;
 using System.Threading;
+using Microsoft.EntityFrameworkCore.Metadata.Internal;
```

Comment:
??? why are we using EF namespace here?

Reply:
No idea. I probably did some code in the past that required it and my IDE automatically added it. I've removed it together with other not used imports.